plugins/bc-widget-area/tests/TestCase/Model/Entity/WidgetAreaTest.php

```diff
@@ -36,7 +36,26 @@ public function test_getCount()
      */
     public function test_getWidgetsArray()
     {
-        $this->markTestIncomplete('こちらのテストはまだ未確認です');
+        $widgetArea = new WidgetArea([
+            'widgets' => 'a:1:{i:0;a:1:{s:4:"test";a:2:{s:4:"sort";i:1;s:4:"name";s:4:"test";}}}',
+        ]);
+        //正常系実行
+        $getWidgetsArray = $this->execPrivateMethod($widgetArea, '_getWidgetsArray', []);
```

Comment:
@thangnnmd cái tham số đầu tiên a gọi sai rồi a ạ. phải là $this->WidgetArea chứ a nhỉ.